pom.xml

```diff
@@ -1063,14 +1076,4 @@
             <url>file://${project.basedir}/lib</url>
         </repository>
     </repositories>
-
-    <dependencyManagement>
-    	<dependencies>
-    		<dependency>
-    			<groupId>org.jmdns</groupId>
```

Comment:
Not a Maven maven, sorry if I get this wrong, but does this mean we're no longer using jmdns?  If so, should we remove it from lib/, build.xml et al?

Reply:
I believe this is just removing the dependency management information.  Jmdns is still listed as a dependency elsewhere in the file after this change.

Reply:
jmdns was in the pom.xml file twice. This simply removes the second entry for it.